packages/test/src/load/setup.ts

```diff
@@ -7,6 +7,7 @@ async function createNamespace(connection: Connection, namespace: string, maxAtt
   for (let attempt = 1; attempt <= maxAttempts; ++attempt) {
     try {
       await connection.service.registerNamespace({ namespace, workflowExecutionRetentionPeriod: msToTs('1 day') });
+      break;
```

Comment:
Apparently this broke CI, the test we have for checking if a namespace is available is inaccurate.

Reply:
Well this line didn't, since it's only load and that's running OK - but the thing in `scripts` apparently doesn't work yeah